ros2_batch_job/__main__.py

```diff
@@ -470,8 +470,8 @@ def run(args, build_function, blacklisted_package_names=None):
         job.run(['"%s"' % job.python, '-m', 'pip', '--version'], shell=True)
         # Install pip dependencies
         pip_packages = list(pip_dependencies)
-        if args.os == 'windows':
-            if not args.packaging and args.cmake_build_type and args.cmake_build_type.casefold() == 'debug':
+        if sys.platform == 'win32':
+            if not args.packaging and args.cmake_build_type and args.cmake_build_type == 'Debug':
```

Comment:
Why should this be conditional on `not args.packaging`?

#272 removes that part from the condition.